flake8_import_order/checker.py

```diff
@@ -59,20 +58,18 @@ def check_order(self):
             if not pycodestyle.noqa(self.lines[import_.lineno - 1]):
                 imports.append(import_)
 
-        if style_option == 'cryptography':
-            style = Cryptography(imports)
-        elif style_option == 'google':
-            style = Google(imports)
-        elif style_option == 'pep8':
-            style = PEP8(imports)
-        elif style_option == 'smarkets':
-            style = Smarkets(imports)
-        elif style_option == 'appnexus':
-            style = AppNexus(imports)
-        elif style_option == 'edited':
-            style = Edited(imports)
-        else:
+        try:
+            style_entry_point = next(
```

Comment:
You can probably pass the entry-point through to here on the options object so you don't have to delay style loading until here. This also means you're not loading it for each and every file.